exe/matrix_from_metadata_v2

```diff
@@ -13,7 +13,7 @@ IMAGE_TABLE = {
   'Windows-2012 R2' => 'windows-2012-r2-core',
   'Windows-2016' => 'windows-2016',
   'Windows-2019' => 'windows-2019-core',
-  'Windows-2022' => 'windows-2022',
+  # 'Windows-2022' => 'windows-2022', Image not yet availible and no date given for when it will be
```

Comment:
```suggestion
  # 'Windows-2022' => 'windows-2022', Image not yet available in GCP/Cloud-CI; no date on availability, but this can be uncommented then
```